beacon-light-client/plonky2/package.json

```diff
@@ -27,7 +27,7 @@
     "chalk": "^4.1.2",
     "circomlibjs": "^0.1.7",
     "colors": "^1.4.0",
-    "ethers": "^5.7.0",
+    "ethers": "^6.11.1",
```

Comment:
Are you sure this upgrade is necessary? It would be better if we could keep the same version of `ethers` across the whole project.